### PR TITLE
Use parsed includeDefaultStyles instead of direct input

### LINKED ISSUE #1128: After 2.1.0 update - ERROR: Unable to exclude default styles
I was getting the same default style bug on clean build as described here [#1115](https://github.com/SC5/sc5-styleguide/issues/1115)

I updated to 2.1.0 because it is suppose to have a fix for this issue, however once I update to 2.1.0 I receive the following error when I run gulp -
`ERROR: Unable to exclude default styles `

I thought it might be related to the `includeDefaultStyles` but I am not using this and adding it to my gulp file doesn't seem to effect the outcome. Any help would be greatly appreciated because I am having to revert back to version 1.7.0 to not have the default style bug issue.

Also, I found that if you run gulp twice after a clean build the style bug seems to fix itself. Not sure why?

### PR DESCRIPTION
**Pull request by kasparsd**
Ready for code review.

Fixes #1128 and #1115.

### CODE REVIEW

File: lib/styleguide.js [hunk 1]
```diff
@@ -254,15 +254,6 @@ function filterFiles(files, filter) {
 }
 
 module.exports.generate = function(options) {
-  try {
-    if (options.includeDefaultStyles) {
```
Comment (kasparsd) [inline lib/styleguide.js]: At this point `options.includeDefaultStyles` hasn't gone through `common.sanitizeOptions()` so the default `includeDefaultStyles` is `false`.

**Comment by mehigh** [comment]
Great job, @kasparsd !

**Comment by kasparsd** [comment]
Is there a way to get this merged and published to NPM?

**Comment by junaidrsd** [comment]
@kasparsd sorry for the delay but as @varya already mentioned we're looking for people who can maintain this project. Let us know if you're interested.

**Comment by kasparsd** [comment]
@junaidrsd Totally understand. Sure, I would love to help keep it updated and working.

**Comment by varya** [comment]
@kasparsd Thank you. Great to hear that you want to contribute. I've invited you as a collaborator.

**Pull request merged**